src/js/charts/point.js

```diff
@@ -195,18 +195,43 @@ function mg_color_point_mouseover({color_accessor, scalefns}, elem, d) {
         .attr('cx', args.scalefns.xoutf)
         .attr('cy', d => args.scalefns.youtf(d));
 
+      let highlights
```

Comment:
Missing semicolon here (not sure why the linter didn't pick this up, need to look into that)